jgrapht-core/src/test/java/org/jgrapht/alg/decomposition/NiceTreeDecompositionTestUtil.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * (C) Copyright 2018-2018, by Ira Justus Fesefeldt and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.decomposition;
+
+
+import java.util.*;
+import java.util.Map.*;
+
+import org.jgrapht.*;
+import org.jgrapht.graph.*;
+
+/**
+ * Tests for the {@link NiceDecompositionBuilder}
+ *
+ * @author Ira Justus Fesefeldt (PhoenixIra)
+ */
+public final class NiceTreeDecompositionTestUtil
+{
+    //static class
+    private NiceTreeDecompositionTestUtil() {}
+    
+    /**
+     * Tests whether the tree decomposition, consisting of the {@code decomposition} and the {@code map} 
+     * with the given {@code root} is nice.
+     * 
+     * @param decomposition the tree decomposition to check
+     * @param map the map of the tree decomposition
+     * @param root the root of the tree decomposition
+     */
+    public static <V,E,W> boolean isNiceDecomposition(Graph<V,E> decomposition, Map<V,Set<W>> map, V root){
+        
+        //empty graph
+        if(decomposition.vertexSet().isEmpty())
+            return true;
+        
+        Queue<V> queue = new ArrayDeque<>();
+        
+        //test and add root
+        if(map.get(root).size() != 1)
```

Comment:
I think we should use braces with the `if` statements when their body consists of a single line. So, please, add braces in the single line `if` statements.